pinot-core/src/main/java/org/apache/pinot/core/io/writer/impl/FixedBitSVForwardIndexWriter.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.io.writer.impl;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteOrder;
+import org.apache.pinot.core.io.util.FixedBitIntReaderWriter;
+import org.apache.pinot.core.io.writer.ForwardIndexWriter;
+import org.apache.pinot.core.segment.memory.PinotDataBuffer;
+
+
+/**
+ * Bit-compressed dictionary-encoded forward index writer for single-value columns. The values written are dictionary
+ * ids.
+ */
+public class FixedBitSVForwardIndexWriter implements ForwardIndexWriter {
```

Comment:
Why not just change the name of existing file FixedBitSingleValueWriter and remove all the unsupported operations. ? You are already doing that for FixedBitMVForwardIndexWriter which was originally FixedBitMultiValueWriter

Reply:
I don't quite follow. `FixedBitSingleValueWriter` is already renamed to `FixedBitSVForwardIndexWriter`

Reply:
I was suggesting that instead of deleting FixedBitSingleValueReader and creating FixedBitSVForwardIndexReader, you can just rename the file  (and do the cleanup) since you have followed this process for FixedBitMultiValueReader. You have renamed it instead of deleting and creating a new file. Same is applicable for FixdBitSvForwardIndexWriter.

Reply:
That is what I did actually, but git might identify it as a new file because the percentage of change is high (the class itself is fairly small)